api/types/externalauditstorage/config/config.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2024 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// IntegrationConfExternalAuditStorage contains the arguments of the
+// `teleport integration configure externalauditstorage` command
```

Comment:
Does this really belong in the API module?

Reply:
Updated the type name and its godoc.

Reply:
I mean I'm not convinced it belongs anywhere under `api/`..

`teleport integration configure` is inherently Teleport-specific and not something I'd expect third-party users of our API to need.

Reply:
Yeah, looks better.
Thank you